build/cqs.tsv

```diff
@@ -66,6 +71,9 @@ io.dropwizard.metrics:metrics-core	3.1.2	compile
 io.dropwizard.metrics:metrics-ganglia	3.1.2	compile
 io.dropwizard.metrics:metrics-graphite	3.1.2	compile
 io.netty:netty-all	4.0.23.Final	compile
+io.netty:netty-buffer	4.0.41.Final	compile
+io.netty:netty-common	4.0.41.Final	compile
+io.netty:netty-handler	4.0.41.Final	compile
```

Comment:
I think I tried to use the same version we already have, but it died... I am planning to go back and check all these again